src/ekka_rlog_activity.erl

```diff
@@ -44,6 +45,15 @@ ro_transaction(Fun) ->
     assert_ro(),
     Ret.
 
+-spec clear_table(ekka_rlog_lib:table()) -> ok.
+clear_table(Tab) ->
+    case get(mnesia_activity_state) of
```

Comment:
The two clauses are a no-op, but if we ever call this outside of an transaction it will yield a case clause.